config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_accesscontextmanageraccesslevels.accesscontextmanager.cnrm.cloud.google.com.yaml

```diff
@@ -286,59 +279,54 @@ spec:
                 - conditions
                 type: object
               custom:
-                description: |-
-                  Custom access level conditions are set using the Cloud Common Expression Language to represent the necessary conditions for the level to apply to a request.
-                  See CEL spec at: https://github.com/google/cel-spec.
+                description: A `CustomLevel` written in the Common Expression Language.
                 properties:
                   expr:
-                    description: |-
-                      Represents a textual expression in the Common Expression Language (CEL) syntax. CEL is a C-like expression language.
-                      This page details the objects and attributes that are used to the build the CEL expressions for
-                      custom access levels - https://cloud.google.com/access-context-manager/docs/custom-access-level-spec.
+                    description: Required. A Cloud CEL expression evaluating to a
+                      boolean.
                     properties:
                       description:
-                        description: Description of the expression.
+                        description: Optional. Description of the expression. This
+                          is a longer text which describes the expression, e.g. when
+                          hovered over it in a UI.
                         type: string
                       expression:
                         description: Textual representation of an expression in Common
                           Expression Language syntax.
                         type: string
                       location:
-                        description: String indicating the location of the expression
-                          for error reporting, e.g. a file name and a position in
-                          the file.
+                        description: Optional. String indicating the location of the
+                          expression for error reporting, e.g. a file name and a position
+                          in the file.
                         type: string
                       title:
-                        description: Title for the expression, i.e. a short string
-                          describing its purpose.
+                        description: Optional. Title for the expression, i.e. a short
+                          string describing its purpose. This can be used e.g. in
+                          UIs which allow to enter the expression.
                         type: string
-                    required:
-                    - expression
```

Comment:
(It is easy to add a // +required comment here, though TBH we don't think it really matters)